config/metaconfig.php

```diff
@@ -3,6 +3,10 @@
 $plugin_mcf['advancedform']['mail_captcha']="bool";
 $plugin_mcf['advancedform']['mail_type']="enum:html,text";
 $plugin_mcf['advancedform']['mail_confirmation']="bool";
+$plugin_mcf['advancedform']['smtp_enabled']="bool";
+$plugin_mcf['advancedform']['smtp_port']="xenum:25,587,465";
+$plugin_mcf['advancedform']['smtp_tls']="enum:,STARTTLS,SMTPS";
```

Comment:
https://github.com/cmsimple-xh/cmsimple-xh/commit/1185d40a5986a0ef4a88175eb1283ce496282f4a

`$plugin_mcf['advancedform']['smtp_password']="password";`

Reply:
Ah, nice! Thank you.